<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectMainWidget</class>
 <widget class="QWidget" name="DetectMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnFire">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ğŸ”¥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMap">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ğŸ“Š</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSync">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SyncSynchronizing"/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>642</width>
     <height>434</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="videoLayout">
    <item>
     <widget class="QLabel" name="videoLabel">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>360</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startCameraButton">
      <property name="text">
       <string>Start Camera</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnIntrusion">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ğŸšª</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGrowth">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ğŸ“</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
